docs/4.4/api-reference.md

```diff
@@ -54,14 +54,105 @@ certificate lifetime.
 
 # gRPC APIs
 
+## Client QuickStart
+
+```go
+{!examples/go-client/client.go!}
+```
+
 ## Audit Events API
 Coming Soon
 
 ## Certificate Generation API
 Coming Soon
 
+## Roles APIs
+
+[Roles](http://localhost:6600/enterprise/ssh-rbac/#roles) are used to define what resources and actions a user is allowed/denied access to.
+
+#### Create a new role
+
+```go
+import github.com/gravitational/teleport/lib/services
+
+// creates a new dev role which can log into teleport as 'teleport'
+roleSpec := services.RoleSpecV3{
+	Allow: services.RoleConditions{
+    Logins: []string{"teleport"},
+	},
+}
```

Comment:
```suggestion
roleSpec := services.RoleSpecV3{
  Allow: services.RoleConditions{
    Logins: []string{"teleport"},
  },
}
```

Reply:
Hmm I already made this change, not sure why is showing up like this on github. Maybe my tabs aren't converting to spaces properly.